src/jsonld/event.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FC } from 'react';
+import Head from 'next/head';
+
+import markup from '../utils/markup';
+import formatIfArray from '../utils/formatIfArray';
+import { Address } from '../types';
+import buildAddress from '../utils/buildAddress';
+
+type Location = {
+  name: string;
+  address: Address;
+  sameAs?: string;
+};
+
+export interface EventJsonLdProps {
+  name: string;
+  startDate: string;
+  endDate: string;
+  location: Location;
+  url?: string;
+  description?: string;
+  images?: string[];
+}
+
+const buildLocation = (location: Location) => `
```

Comment:
Because `location.sameAs` is optional I think the JSON will be invalid when due to the trailing `,` from `buildAddress()` above it. If you move name to be last (and remove `,`)  and give sameAs a trailing comma it should be ok I think. So it would look like: 

```
const buildLocation = (location: Location) => `
  "location": {
    "@type": "Place",
    ${buildAddress(location.address)}
    ${location.sameAs ? `"sameAs": "${location.sameAs}",` : ``}
    "name": "${location.name}"
  },
`;
```